gradle/code-coverage.gradle

```diff
@@ -11,6 +11,13 @@ apply plugin: 'jacoco'
 repositories {
   mavenCentral()
   gradlePluginPortal()
+  // TODO: Find the way to use the repositories from RepositoriesSetupPlugin
```

Comment:
@nknize @dblock I don't have better solution at the moment but this fix unblocks the builds